lib/Auth/Process/DatabaseCommand.php

```diff
@@ -23,10 +23,43 @@ public function __construct()
         $this->dbDriver = $this->databaseConnector->getDbDriver();
         $this->statisticsTableName = $this->databaseConnector->getStatisticsTableName();
         $this->detailedStatisticsTableName = $this->databaseConnector->getDetailedStatisticsTableName();
+        $this->ipStatisticsTableName = $this->databaseConnector->getIpStatisticsTableName();
         $this->identityProvidersMapTableName = $this->databaseConnector->getIdentityProvidersMapTableName();
         $this->serviceProvidersMapTableName = $this->databaseConnector->getServiceProvidersMapTableName();
     }
+    private function insertIpLogins($userId, $ip, $year, $month,$day, $time, $sourceIdp, $service)
```

Comment:
rename function to "writeLoginIp" to be consistent with the function "writeLogin"

Reply:
Also, let's keep the parameters of the function in the same order as 
`writeLogin($year, $month, $day, $sourceIdp, $service, $user = null)`
and put the new parameters in the end